src/main/java/org/folio/service/util/EventHandlingUtil.java

```diff
@@ -62,22 +63,20 @@ public static Future<Boolean> sendEventToKafka(
     String recordId = extractHeader(kafkaHeaders, "recordId");
 
     String producerName = eventType + "_Producer";
-    KafkaProducer<String, String> producer =
-      KafkaProducer.createShared(vertx, producerName, kafkaConfig.getProducerProps());
-    producer.write(record)
+
+    KafkaProducer<String, String> producer = new SimpleKafkaProducerManager(vertx, kafkaConfig).createShared(eventType);
```

Comment:
The same regarding defining `"_Producer"` explicitly

Reply:
You mean, producerName which locates on line 65?
Because it is only used in logging

Reply:
Oh ok, it just has been implemented in different ways initially. So now `producerName` could be inlined since it's used only for logging

Reply:
yeah, that would be great to replace it with in-lined version. However, it points a little bit more clarification since it is a producerName